src/frontends/onnx/tests/tests_python/test_backend.py

```diff
@@ -683,6 +686,12 @@ def expect_fail(test_case_path, xfail):  # type: (str) -> None
     ),
 ]
 
+# if platform == 'darwin':
+#     tests_expected_to_fail.append((
+#         xfail_issue_58676,
+#         "OnnxBackendNodeModelTest.test_div_uint8_cpu"
+#     ))
```

Comment:
let's also remove ignorance of test_onnx/test_backend.py from other workflows for Python API 1.0